src/core/shape/vertex.js

```diff
@@ -609,21 +610,86 @@ p5.prototype.endContour = function() {
  * </code>
  * </div>
  *
+ * @example
+ * <div>
+ * <code>
+ * let fx;
+ * let vs = `#version 300 es
+ *
+ * precision mediump float;
+ *
+ * in vec3 aPosition;
+ *
+ * uniform mat4 uModelViewMatrix;
+ * uniform mat4 uProjectionMatrix;
+ *
+ * void main() {
+ *
+ *   vec4 positionVec4 = vec4(aPosition, 1.0);
+ *   gl_Position = uProjectionMatrix * uModelViewMatrix * positionVec4 -
+ *   vec4(vec2(float(gl_InstanceID) * 100.0, 0.0), 0.0, 0.0);
+ * }
+ * `;
+ * let fs = `#version 300 es
+ *
+ * precision mediump float;
+ *
+ * out vec4 outColor;
+ *
+ * void main() {
+ *   outColor = vec4(255, 0, 0, 255);
+ * }
+ * `;
+ *
+ * function setup() {
+ *   createCanvas(400, 400, WEBGL);
+ *   // Please note that it's best practice to put shaders in other files
```

Comment:
I think this is a stylistic choice, we can remove this comment.

Reply:
alr